containers/unit-tests/setup.sh

```diff
@@ -7,9 +7,10 @@ dependencies="\
     autoconf \
     automake \
     build-essential \
-    chromium-browser \
-    clang python3 \
+    chromium \
+    clang \
     curl \
+    debian-archive-keyring \
```

Comment:
Why is this necessary?  amd64?  debug?  both?

We install this only as part of `${dependencies}`, which is the last thing we install, so it's only installed after it might have been useful... Is this really necessary at all, or did I miss something?

Reply:
Well spotted, this is a leftover from the "move to ubuntu 20.04" variant, which needed to pull chromium from Debian. Removed.